lib/rewriteUrl.js

```diff
@@ -1,6 +1,12 @@
 function interpolateIdIfAvailable(url) {
```

Comment:
Nope, no reason. It started so simply but be would be a good idea to use a lib for this. How about using built-in libs?

```
const url = require('url');
const qs = require('querystring');

const toParse = 'http://www.nhs.uk/Services/gp/Overview/DefaultView.aspx?id=35956';

const query = url.parse(toParse).query;

const id = qs.parse(query).id;
```

https://runkit.com/st3v3nhunt/58d8fb5454c1be00146d0d91